doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+Each of the following methods defines local feature attribution slightly differently. In both, however, we assign attribution values to each feature to indicate how significant those features were in making the model prediction.
+
+Let $f:\mathbb{R}^n \rightarrow \mathbb{R}$. $f$ might be a regression, a single component of a multi regression or a probability of a class in a classification model. If $x=(x_1,... ,x_n) \in \mathbb{R}^n$ then an attribution of the prediction at input $x$ is a vector $a=(a_1,... ,a_n) \in \mathbb{R}^n$ where $a_i$ is the contribution of $x_i$ to the prediction $f(x)$.
+
+The attribution values should satisfy specific properties:
+
+1. Efficiency/Completeness: The sum of attributions equals the difference between the prediction and the baseline/average. We're interested in understanding the difference each feature value makes in a prediction compared to some uninformative baseline.
+2. Symmetry: If the model behaves the same after swapping two variables $x$ and $y$, then $x$ and $y$ have equal attribution. If this weren't the case, we would be biasing the attribution towards certain features over other ones.
+3. Dummy/Sensitivity: If a variable does not change the output of the model, then it should have attribution 0. If this were not the case, we'd be assigning value to a feature that provides no information.
+4. Additivity/Linearity: The attribution for a feature $x_i$ of a linear composition of two models $f_1$ and $f_2$ given by $c_1 f_1 + c_2 f_2$ is $c_1 a_{1, i} + c_2 a_{2, i}$ where $a_{1, i}$ and $a_{2, i}$ is the attribution for $x_1$ and $f_1$ and $f_2$ respectively.
+
+#### Integrated Gradients
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| TF/Kera     | Classification | Tabular     |
+|             | Regression     | Image       |
+|             |                | Text        |
+|             |                | Categorical |
+
+
+This method computes the attribution of each feature by integrating the model partial derivatives along a path from a baseline point to the instance. Let $f$ be the model and $x$ the instance of interest. If $f:\mathbb{R}^{n} \rightarrow \mathbb{R}^{m}$ where $m$ is the number of classes the model predicts then let $F=f_k$ where $k \in \{1,..., m\}$. If $f$ is single-valued then $F=f$. We also need to choose a baseline value, $x'$.
```

Comment:
add a sentence similar to "Then the integrated gradients attributions are defined as"